examples/rp-process/rp_process.py

```diff
@@ -48,18 +47,20 @@
 core_nuclei = list(map(Nucleus, core_nuclei))
 core_lib = full_lib.linking_nuclei(core_nuclei)
```

Comment:
we don't need this map -- `linking_nuclei` can take a list of string names

Reply:
It's also used later for `seeds`.

Reply:
ah okay